srtcore/core.cpp

```diff
@@ -3844,6 +3843,15 @@ EConnectStatus CUDT::postConnect(const CPacket& response, bool rendezvous, CUDTE
 
     setupCC();
 
+    // Update timers
+    uint64_t currtime_tk;
+    CTimer::rdtsc(currtime_tk);
+    m_ullLastRspTime_tk    = currtime_tk;
+    m_ullNextACKTime_tk    = currtime_tk + m_ullSYNInt_tk;
+    m_ullNextNAKTime_tk    = currtime_tk + m_ullNAKInt_tk;
+    m_ullLastRspAckTime_tk = currtime_tk;
+    m_ullLastSndTime_tk    = currtime_tk;
+
```

Comment:
I think the same should be done also around `acceptAndRespond`. This one is for caller, the accepted-off socket may suffer the same problem.

Reply:
Good point!